fortran_playground/fortran_playground.py

```diff
@@ -0,0 +1,94 @@
+from sphinx.directives.code import *
```

Comment:
explicit imports of what is needed would be nicer

Reply:
@zmoon Here we could have used the explicit imports but in order to achieve **Exactly similar**  performance of the codeblocks directive , i had to import the entire code.py of sphinx.directives

Reply:
looks like maybe 4 are used: `CodeBlock, parselinenos, dedent_lines, container_wrapper`

Reply:
Thanks